torch/distributed/distributed_c10d.py

```diff
@@ -428,6 +429,9 @@ def init_process_group(backend,
             might result in subsequent CUDA operations running on corrupted
             data. Only one of these two environment variables should be set.
         group_name (str, optional, deprecated): Group name.
+        deviceId (int, optional): device Id in case of NCCL backend
```

Comment:
This can be removed now that there are no changes to init process group right?

Reply:
thanks for reminding.